docs/Using Any API/contract-addresses.md

```diff
@@ -20,7 +20,7 @@ For a comprehensive list of data providers, oracles, jobs, adapters, and more, v
 >
 > We strongly recommend that you deploy to a [testnet](#testnets) first, then move to mainnet.
 
-LINK address:  [`0x514910771af9ca656af840dff83e8264ecf986ca`](https://etherscan.io/token/0x514910771af9ca656af840dff83e8264ecf986ca)
+LINK address: <a class="erc-token-address" id="1_0x514910771af9ca656af840dff83e8264ecf986ca" href="https://etherscan.io/token/0x514910771af9ca656af840dff83e8264ecf986ca" target="_blank" rel="noreferrer, noopener">`0x514910771af9ca656af840dff83e8264ecf986ca`</a>
```

Comment:
We already add noreferrer noopener by JS I believe. Can you double check if we need this? I don't think we do.

Reply:
@StephenFluin  we don't. For instance, open "_site/docs/deploy-your-first-contract/index.html" and you won't find noreferrer noopener on external links. We can look for a Eleventy plugin that does that by default for external links

Reply:
resolved